src/OrchardCore.Modules/OrchardCore.OpenId/OrchardCore.OpenId.csproj

```diff
@@ -8,6 +8,9 @@
 
     Provides OpenId Connect client, server and management features.</Description>
     <PackageTags>$(PackageTags) OrchardCoreCMS OpenId</PackageTags>
+    
+    <!-- Required to build on net10.0 with validation interceptors until it's part of the SDK -->
+    <InterceptorsNamespaces>$(InterceptorsNamespaces);Microsoft.AspNetCore.Http.Validation.Generated</InterceptorsNamespaces>
```

Comment:
Why this is need in the current code base?

Reply:
We're adding a new source generator in the shared framework to support type validation. Part of its functionality uses source generators that emit interceptors. The compiler requires this property to help with discovery and consent from the user. Typically, we auto-apply it in the SDK so the user doesn't pay the cost.

@sebastienros Extra thought I had after our convo yesterday: the generator only intercepts "AddValidation" calls so there is is a chance that there might be one in this source code that it is greedily discovering.

Reply:
Thanks for you clarification Safia

Reply:
@captainsafia the only `.AddValidation()` call here is the one used to register and configure the OpenIddict validation handler and its ASP.NET Core authentication integration: if there's a bug affecting `.AddValidation()` calls, it won't just affect Orchard but basically all the projects that use OpenIddict, including pretty much everything based on ABP Framework or Umbraco (so... a lot of projects 🤣).

Do you have a ticket in dotnet/aspnetcore that tracks this issue?

Reply:
> Do you have a ticket in dotnet/aspnetcore that tracks this issue?

Found it: https://github.com/dotnet/aspnetcore/issues/61177.